docs/team/zhongfu.md

```diff
@@ -5,32 +5,48 @@ title: Zhongfu's Project Portfolio Page
 
-* **Project management**:
-  * tbd
+* **Team tasks**:
+  * Handled initial triage for PE-D issues, and followed up on PRs and old issues
+  * Reviewed [24 pull requests](https://github.com/AY2122S2-CS2103T-T11-4/tp/pulls?q=reviewed-by%3Azhongfu)
 
-* **Enhancements to existing features**:
-  * tbd
+* **Enhancements to existing feature**:
+  * New serializer/deserializer architecture for model classes [#65](https://github.com/AY2122S2-CS2103T-T11-4/tp/pull/65), [#88](https://github.com/AY2122S2-CS2103T-T11-4/tp/pull/88)
+    * What it does: (De)serialize model classes to/from JSON with custom (de)serializers (bundled together with the model classes, as nested classes).
+    * Justification: The previous (de)serialization mechanism is not as intuitive for new developers, as the relevant classes are spread across multiple packages and types reused in multiple classes cannot be automatically (de)serialized. It also makes implementing novel features (e.g. backward-compatibility) more difficult. This enhancement aims to alleviate some of those issues by allowing for increased flexibility, and by bundling the deser classes with the associated object classes.
+    * Highlights: Deciding on an architecture that serves all our existing needs (and allows for easy extensibility) was somewhat complicated, as Jackson is a complex library that (as a result) presents many ways to perform any given task.
+  * GUI bug fixes: Allow window size to increase/decrease w/ elements resizing to fit [#218](https://github.com/AY2122S2-CS2103T-T11-4/tp/pull/218), tag wrapping [#209](https://github.com/AY2122S2-CS2103T-T11-4/tp/pull/209)
+  * Quality-of-life bug fixes (e.g. email regex changes [#69](https://github.com/AY2122S2-CS2103T-T11-4/tp/pull/69), `Desktop.browse()` hanging on Linux [#208](https://github.com/AY2122S2-CS2103T-T11-4/tp/pull/208))
 
 * **Documentation**:
   * User Guide:
-    * tbd
+    * Changes to reflect feature changes and fix breakages (e.g. [#200](https://github.com/AY2122S2-CS2103T-T11-4/tp/pull/200))
   * Developer Guide:
-    * tbd
+    * Explain serdes architecture and write tutorials on writing new serdes classes [#87](https://github.com/AY2122S2-CS2103T-T11-4/tp/pull/87), [#89](https://github.com/AY2122S2-CS2103T-T11-4/tp/pull/89)
 
 * **Community**:
-  * tbd
+  * Participated in [13 forum discussions](https://github.com/nus-cs2103-AY2122S2/forum/issues?q=commenter%3Azhongfu)
+  * Submitted [11 bugs](https://github.com/AY2122S2-CS2103T-T13-2/tp/issues?q=zhongfu) during the PE dry-run, 10 of which were accepted as-is (except for a [very-low severity bug](https://github.com/AY2122S2-CS2103T-T13-2/tp/issues/169) eventually marked as a documentation bug)
 
 * **Tools**:
-  * tbd
+  * Added git pre-commit hooks that enforce correct style and all tests passing, as well as fix common issues (e.g. incorrect FXML versions caused by SceneBuilder [#212](https://github.com/AY2122S2-CS2103T-T11-4/tp/pull/212))
```

Comment:
yasss thanks for this addition we really benefitted from it. I think it can go under your Team Tasks section.

```suggestion
  * Added git pre-commit hooks that enforce correct style and all tests passing, as well as fix common issues (e.g. incorrect FXML versions caused by SceneBuilder [\#212](https://github.com/AY2122S2-CS2103T-T11-4/tp/pull/212))
```